package.json

```diff
@@ -129,13 +129,13 @@
     "jest-styled-components": "^7.1.1",
     "lint-staged": "^10.5.4",
     "mini-css-extract-plugin": "^1.6.2",
-    "msw": "^0.35.0",
-    "nodemon": "^2.0.22",
+    "msw": "^2.6.3",
```

Comment:
Ideally, we want to avoid any changes to `package.json` and `package-lock.json` unless absolutely necessary. I don't believe those changes are needed here, so it would be best to remove them from this PR!